kafka_exporter.go

```diff
@@ -511,7 +511,7 @@ func (e *Exporter) collect(ch chan<- prometheus.Metric) {
 		}
 	}
 
-	N := minx(len(topics)/2, e.topicWorkers)
+	N := minx(len(topics), e.topicWorkers)
```

Comment:
@echernous If I understand the problem you are reporting correctly, you are highlighting the case where when `len(topics)==1`, there are no `loopTopics()` go routinues started.  The intent of your change is to ensure that in this case, loopTopics is still run.

I'm unsure whether changing your solution of changing `len(topics)/2` => `len(topics)` and altering the default is the best one.  That may disturb behaviour others are relying on.

I'm wondering about: `N := minx(math.Ceil(len(topics)/2.0), e.topicWorkers)`

Reply:
thanks @k-wall , yeah it's better to avoid breaking/disturbing changes, applied your solution

Reply:
it ended up too messy to read with explicit castings `N := minx(int(math.Ceil(float64(len(topics))/float64(2))), e.topicWorkers)`, replaced with an extra if statement. It probably would be better to repurpose the minx functions into getTopicWorkers but I'm not sure if there are any future plans to use the generic minx